docs/serial_driver.md

```diff
@@ -181,7 +181,7 @@ The `SERIAL` Subsystem is supported for the majority of ChibiOS MCUs and should
 #define HAL_USE_SERIAL TRUE
 ```
 
-2. In your keyboards `mcuconf.h`: activate the USART peripheral that is used on your MCU. The shown example is for an STM32 MCU, so this will not work on MCUs by other manufacturers. You can find the correct names in the `mcuconf.h` files of your MCU that ship with ChibiOS. 
+2. In your keyboards `mcuconf.h`: activate the USART peripheral that is used on your MCU. The shown example is for an STM32 MCU, so this will not work on MCUs by other manufacturers. You can find the correct names in the [`mcuconf.h` files of your MCU that ship with ChibiOS](https://github.com/ChibiOS/ChibiOS-Contrib). 
```

Comment:
These values come from a magnitude of files, eg. in [platforms/chibios/boards](https://github.com/qmk/qmk_firmware/tree/master/platforms/chibios/boards).
This also isn't the only place this would need modifying.

Reply:
Nice! Thanks for the link. This is the first reference I've come across in the docs. I don't know the other places you allude to.

Would you rather link this in a note with what you said?